.travis.yml

```diff
@@ -0,0 +1,32 @@
+sudo: required
+dist: trusty
+language: generic
+
+notifications:
+  email:
+    on_success: always
+    on_failure: always
+#    recipients:
+#      - jane@doe
+
+env:
+  matrix:
+    - ROS_DISTRO="indigo"   ROS_REPO=ros               UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO="indigo"   ROS_REPO=ros-shadow-fixed  UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO="kinetic"  ROS_REPO=ros               UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO="kinetic"  ROS_REPO=ros-shadow-fixed  UPSTREAM_WORKSPACE=file
+
+matrix:
+  allow_failures:
+    - env: ROS_DISTRO="indigo"  ROS_REPO=ros               UPSTREAM_WORKSPACE=file
+    - env: ROS_DISTRO="indigo"  ROS_REPO=ros-shadow-fixed  UPSTREAM_WORKSPACE=file
+
+install:
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+
+env:
```

Comment:
hm, I assumed it was a 'merge dictionary' thing. Let's see if this works better.